src/bin/club-find

```diff
@@ -39,6 +46,9 @@ module.exports = program = program
 spin.setSpinnerString(27);
 
 const main = async () => {
+    if (program.args.length) {
+        log('Using search story API since search operators were passed ...');
```

Comment:
Can we have this log message respect the `--quiet` flag?

Reply:
@andjosh updated.